docs/how-to/index.rst

```diff
@@ -39,6 +39,16 @@ compiled a couple of guides for you to facilitate your contribution.
 
    documentation <documentation/index>
 
+Extensions
+----------
+
+Learn how to use framework and platform extensions in your project.
+
+.. toctree::
+   :maxdepth: 1
+
+   Configure the build base for ExpressJS App <configure-the-build-base-for-expressjs-app>
```

Comment:
This line is too long and has a typo ("an" ExpressJS App). Can we just use the target page's title?